### DISCUSSION TITLE
JCL installation problem.

### DISCUSSION THREAD

**Discussion by DilanRona** [project-jedi/jcl#145] [Q&A]
Unable to install JCL. Tried via command line, after downloading from git, and following the instructions. Tried via Delphi 12 GetIT Package manager. Both resulted in the same error.
`
jcl\install
Initializing/Updating git submodules...

===================================================================
Compiling JediIncCheck...
Using Embarcadero RAD Studio 10.6

This version of the product does not support command line compiling.

An error occured while compiling the installer. Installation aborted.

Press any key to continue . . .
`

Tried the community version of Delphi, and the trial version. And I cannot compile WinBox for 86Box because of this problem with me being unable to install JCL.

**Comment by obones** [comment]
As the message indicates: 

> This version of the product does not support command line compiling.

Neither the community version, nor the trial version provide the command line compiler (dcc32) that is used by the JCL installer.

You have to resort to manual installation that is described in the [Readme.txt](https://github.com/project-jedi/jcl/blob/master/jcl/docs/Readme.txt) file.

**Comment by DilanRona** [comment]
You are making something that initially started out as a simple install so overtly complicated that it is giving me a headache. And I am getting frustrated now.

**Reply by ronaldhoek** [reply]
Basically, it has nothing to do with the installer, but with the product version (Delphi) you are using (as the error indicates).

But @obones it would be a good idea to mention this in the readme of the projects!

**Reply by obones** [reply]
It is already mentioned, I even gave the link to the readme file. Sure, the process is quite tiresome, but we've always been clear: we don't support free versions because Embarcadero does not care that we do. Trust me, we tried to convince them...

**Reply by ronaldhoek** [reply]
so the suggestion in https://github.com/project-jedi/jcl/pull/147 is not needed ..?

**Reply by obones** [reply]
It's nice, but it should also point to the Readme.txt file for manual installation.
Note that I know it's a cumbersome process to install manually, but there's nothing we can do to make this any easier.

**Reply by ronaldhoek** [reply]
Updated the PR to mention the readme.ex in \jcl\docs\
